pinot-broker/src/main/java/org/apache/pinot/broker/api/HttpRequesterIdentity.java

```diff
@@ -30,6 +31,17 @@ public class HttpRequesterIdentity extends RequesterIdentity {
   private Multimap<String, String> _httpHeaders;
   private String _endpointUrl;
 
+  public static HttpRequesterIdentity fromRequest(org.glassfish.grizzly.http.server.Request request) {
```

Comment:
nit: import `org.glassfish.grizzly.http.server.Request` instead of specifying the full path here?